ArduPlane/mode_autoland.cpp

```diff
@@ -72,7 +82,7 @@ bool ModeAutoLand::_enter()
         gcs().send_text(MAV_SEVERITY_WARNING, "Takeoff initial direction not set");
         return false;
     }
-
+    climb_before_turn_alt = (climb_before_turn_alt < final_wp_alt)? final_wp_alt : climb_before_turn_alt;
```

Comment:
this is directly changing the parameter? I would have expected a compiler error....

Reply:
Looks like you get away with it because both possible values are `AP_Int16` params so its gets the default operator.

Reply:
In anycase, we try not to set params like this if we can avoid it.

Reply:
will assign to another new variable...just trying save memory